src/game/Maps/Map.h

```diff
@@ -221,8 +221,13 @@ class Map : public GridRefManager<NGridType>
         void ChangeMapDifficulty(Difficulty difficulty);
         void SetNewDifficultyCooldown(TimePoint const& newCooldown);
         TimePoint const& GetNewDifficultyCooldown() const { return m_dynamicDifficultyCooldown; }
+        float GetPetKillXPMod() const;
+        float GetQuestXPMod() const;
+        float GetKillXPMod() const;
+        float GetExploreXPMod() const;
 
         MapEntry const* GetEntry() const { return i_mapEntry; }
+        uint32 Expansion() const { return (i_mapEntry) ? i_mapEntry->Expansion() : 0u; }
```

Comment:
@kelbren

Reply:
GetExpansion pretty sure. I know the submethod is called Expansion only but we dont have to copy old codestyle :D

Reply:
Makes sense; name changed accordingly